src/main/scala/net/akmorrow13/endive/utils/Window.scala

```diff
@@ -82,6 +85,14 @@ case class Window(tf: TranscriptionFactors.Value,
   }
 }
 
+
+case class FeaturizedLabeledWindow(labeledWindow: LabeledWindow, features: DenseVector[Double]) {
+
```

Comment:
you use it exactly 5 times, the format of kernelpipeline will be changed anyway to match the notebook stuff